src/main/java/life/qbic/dbase/OpenBisProxy.java

```diff
@@ -86,7 +86,7 @@ private static void init() {
         final ConfigurationManager conf = ConfigurationManagerFactory.getInstance();
         password = conf.getDataSourcePassword();
         username = conf.getDataSourceUser();
-        url = conf.getDataSourceUrl() + "/openbis/openbis" + IApplicationServerApi.SERVICE_URL;
+      url = conf.getDataSourceUrl() + "/openbis/openbis" + IApplicationServerApi.SERVICE_URL;
```

Comment:
```suggestion
        url = conf.getDataSourceUrl() + "/openbis/openbis" + IApplicationServerApi.SERVICE_URL;
```
Please stick to the indentation amount already present in the file.